etcdserver/server_test.go

```diff
@@ -451,6 +451,7 @@ func TestApplyConfChangeError(t *testing.T) {
 		srv := &EtcdServer{
 			r:       raftNode{Node: n},
 			cluster: cl,
+			cfg:     &ServerConfig{},
```

Comment:
Should the cfg be placed at first line? Seems that after r (raftNode) would be suitable for the member. How do you think?